media-gfx/fontforge/fontforge-20220308.recipe

```diff
@@ -0,0 +1,139 @@
+SUMMARY="An outline font editor"
+DESCRIPTION="An outline font editor that lets you create your own OpenType, \
+TrueType, CID-keyed, multi-master fonts and more... or simply edit existing \
+ones."
+HOMEPAGE="https://fontforge.github.io/"
+COPYRIGHT="2000-2018 George Williams
+	The FontForge Project"
+LICENSE="GNU GPL v3
+	BSD (3-clause)"
+REVISION="4"
+SOURCE_URI="https://github.com/fontforge/fontforge/releases/download/$portVersion/fontforge-$portVersion.tar.xz"
+CHECKSUM_SHA256="01e4017f7a0ccecf436c74b8e1f6b374fc04a5283c1d68967996782e15618e59"
+ADDITIONAL_FILES="fontforge.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2 !x86"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	fontforge$secondaryArchSuffix = $portVersion
+	app:fontforge
+	cmd:fontimage$commandSuffix
+	cmd:fontlint$commandSuffix
+	cmd:sfddiff$commandSuffix
+	lib:libfontforge$secondaryArchSuffix = 4 compat >= 4
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgdk_3$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libpython3.10$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libspiro$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libcairo_gobject$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgdk_3$secondaryArchSuffix
+	devel:libgdk_pixbuf_2.0$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libpangocairo_1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libpython3.10$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libspiro$secondaryArchSuffix
```

Comment:
this should probably specify the version as `>= 1` to make sure that it picks up the new version, because the old one was restored to not break the existing version of fontforge. Afterwards we can remove the old version of libspiro again.